digraph "CFG forkotlinx.coroutines.flow.internal.DownstreamExceptionContext.\<init\>(Ljava\/lang\/Throwable;Lkotlin\/coroutines\/CoroutineContext;)V" {
Node_4 [shape=record,label="{4\:\ ?|MTH_ENTER_BLOCK\l}"];
Node_6 [shape=record,label="{6\:\ 0x0000|0x0000: INVOKE  (r0 I:java.lang.Object) type: DIRECT call: java.lang.Object.\<init\>():void A[MD:():void (c)] (LINE:144)\l0x0003: IPUT  \l  (r1 I:java.lang.Throwable A[D('e' java.lang.Throwable)])\l  (r0 I:kotlinx.coroutines.flow.internal.DownstreamExceptionContext)\l kotlinx.coroutines.flow.internal.DownstreamExceptionContext.e java.lang.Throwable  (LINE:145)\l0x0005: IPUT  \l  (r2 I:kotlin.coroutines.CoroutineContext A[D('originalContext' kotlin.coroutines.CoroutineContext)])\l  (r0 I:kotlinx.coroutines.flow.internal.DownstreamExceptionContext)\l kotlinx.coroutines.flow.internal.DownstreamExceptionContext.$$delegate_0 kotlin.coroutines.CoroutineContext  (LINE:147)\l}"];
Node_7 [shape=record,label="{7\:\ 0x0007|RETURN\l|0x0007: RETURN    (LINE:144)\l}"];
Node_5 [shape=record,label="{5\:\ ?|MTH_EXIT_BLOCK\l}"];
MethodNode[shape=record,label="{public void kotlinx.coroutines.flow.internal.DownstreamExceptionContext.\<init\>((r0 'this' I:kotlinx.coroutines.flow.internal.DownstreamExceptionContext A[IMMUTABLE_TYPE, THIS]), (r1 I:java.lang.Throwable A[D('e' java.lang.Throwable), IMMUTABLE_TYPE, METHOD_ARGUMENT]), (r2 I:kotlin.coroutines.CoroutineContext A[D('originalContext' kotlin.coroutines.CoroutineContext), IMMUTABLE_TYPE, METHOD_ARGUMENT]))  | USE_LINES_HINTS\lINLINE_NOT_NEEDED\lDebug Info:\l  \-1 \-0x0007: r1 'e' Ljava\/lang\/Throwable;\l  \-1 \-0x0007: r2 'originalContext' Lkotlin\/coroutines\/CoroutineContext;\l}"];
MethodNode -> Node_4;Node_4 -> Node_6;
Node_6 -> Node_7;
Node_7 -> Node_5;
}

